### ISSUE TITLE
DCGAN Tutorial in TF Learn results different as in the tutoroial

### ISSUE THREAD

**Issue by fadelkoto** [tensorflow/tensorflow#35363]
Hallo. I am a new learner for deep learning. I tried this DCGAN tutorial from Tensor Flow but the training result is different compared to what is shown in the tutorial website. 

This is my result:

![image](https://user-images.githubusercontent.com/36468837/71376946-83e69a80-25c3-11ea-9f77-d9f492a328f0.png)

Based on the tutorial, it supposed to have a result like this:
![image](https://user-images.githubusercontent.com/36468837/71377054-e5a70480-25c3-11ea-8b08-05226e41465a.png)

I already tried to run the train several times after it finishes the 50th epoch. But still it didnt show the same result as the tutorial. I appreciate the help to solve this. Thank you

**Comment by SumanSudhir** [comment]
Hey, there may be something wrong with your training as I trained the same notebook for 50 epochs and getting approx same result as shown in the notebook tutorial
![image](https://user-images.githubusercontent.com/35973532/71428187-38c4a880-26e5-11ea-9aa9-3f8c82053710.png)

**Comment by SumanSudhir** [comment]
Animated preview of the trained model is also as per the shown in notebook
![dcgan](https://user-images.githubusercontent.com/35973532/71428225-75909f80-26e5-11ea-914c-757fe249398d.gif)

**Comment by CleanPegasus** [comment]
Just tried it and the code works as expected. Can you send us the code you tried?

**Comment by ravikyram** [comment]
@fadelkoto 
I have tried on colab with TF version 2.1.0-rc1 and i am getting approximately same result as shown in tutorial. Please, find the gist [here](https://colab.sandbox.google.com/gist/ravikyram/5f6bd200a1a6363e28c1f3bcf64e8850/untitled500.ipynb). Thanks!

**Comment by fadelkoto** [comment]
Hey Guys, thank you for the responses. I checked again my code and I skipped the code for the image normalization:

train_images = (train_images - 127.5) / 127.5 # Normalize the images to [-1, 1]

Thank you.
